build/bazelutil/distdir_files.bzl

```diff
@@ -596,7 +598,7 @@ DISTDIR_FILES = {
     "https://storage.googleapis.com/cockroach-godeps/gomod/github.com/matryer/is/com_github_matryer_is-v1.4.0.zip": "8c49393743a0b5c61b130632e402efa55d4b03456f31aea14a269d47503df112",
     "https://storage.googleapis.com/cockroach-godeps/gomod/github.com/matryer/moq/com_github_matryer_moq-v0.0.0-20190312154309-6cfb0558e1bd.zip": "b9fb2bc3d0894dfaa3cc4298f49c97346ccb66f2f0e6911f4f224ffc9acc3972",
     "https://storage.googleapis.com/cockroach-godeps/gomod/github.com/mattn/go-colorable/com_github_mattn_go_colorable-v0.1.12.zip": "c4637499409a4081fbbd7ea0640aa0af30c0e06a70aa1b207780b0251650d8cd",
-    "https://storage.googleapis.com/cockroach-godeps/gomod/github.com/mattn/go-ieproxy/com_github_mattn_go_ieproxy-v0.0.1.zip": "2a1ad026c677bddef4136c76da19db2204d568b4c2246b2e00a79b47d8c01192",
+    "https://storage.googleapis.com/cockroach-godeps/gomod/github.com/mattn/go-ieproxy/com_github_mattn_go_ieproxy-v0.0.0-20190610004146-91bb50d98149.zip": "2982ad9362d63b30a081fe7609b595fefcc7baaaeda2f5f17a7dbeb087a84020",
```

Comment:
Also a downgrade? And to a custom build? (Also, is this related to azure? something about internet explorer?)

Reply:
yup it's a package azure uses. I guess the new sdk uses an older version of this. it indeed looks like an internet explorer package lol.